widgetry/src/style/mod.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{Color, Text};
+
+pub mod buttons;
+
+#[derive(Clone)]
+pub struct Style {
+    pub outline_thickness: f64,
+    pub outline_color: Color,
+    pub panel_bg: Color,
+    pub hotkey_color: Color,
+    pub hovering_color: Color,
+    pub loading_tips: Text,
+    pub btn_primary_dark: ButtonStyle,
+    pub btn_secondary_dark: ButtonStyle,
+    pub btn_primary_light: ButtonStyle,
+    pub btn_secondary_light: ButtonStyle,
+}
+
+#[derive(Clone)]
+pub struct ButtonStyle {
+    pub fg: Color,
+    pub fg_disabled: Color,
+    pub outline: Color,
+    pub bg: Color,
+    pub bg_hover: Color,
+    pub bg_disabled: Color,
+}
+
+impl Style {
+    pub fn standard() -> Style {
+        Style {
+            outline_thickness: 2.0,
+            outline_color: Color::WHITE,
+            panel_bg: Color::grey(0.4),
+            hotkey_color: Color::GREEN,
+            hovering_color: Color::ORANGE,
+            loading_tips: Text::new(),
+
+            // Buttons
```

Comment:
This is super nice! I'm not particularly concerned yet with letting people provide custom skins for widgetry, but slapping serde around this struct is probably where we could start, once somebody asks.